Amazon.QLDB.Driver.IntegrationTests/utils/IntegrationTestBase.cs

```diff
@@ -429,4 +436,57 @@ public string GetDisplayName(MethodInfo methodInfo, object[] data)
             return null;
         }
     }
+
+    class ParameterObject
+    {
+        public string Name
```

Comment:
We agreed this should not be public

Reply:
It is here so we can get the test running. Will update once the object mapper [PR](https://github.com/amzn/ion-object-mapper-dotnet/pull/36) to handle internal/protected internal properties is merged in.

Reply:
Updated to internal